### DISCUSSION TITLE
Whisper transcription very long

### DISCUSSION THREAD

**Discussion by Marin-dev** [deepgram/community#620] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Hi there, 
I would like to use whisper model for my transcription and the procession time is very long ( about 20 minutes for a 20 second video) - always in pending statut.
Can you explain me why the process time is so long ?

For instance :
```
request id : 00b4f34f-fdff-4cac-bcdf-d7ee189c3867
Post : POST /v1/listen?model=whisper-large&language=fr&smart_format=true&punctuate=true&diarize=true
request : 
    const transcription = await deepgram.listen.prerecorded.transcribeFile(
      fs.createReadStream(localVideoPath),
      {
        model: "whisper-large",
        language: language,
        smart_format: true,
        punctuate: true,
        diarize: true,
      }
    );

```

I tried to use medium model or tiny but it's still very long
Thanks you

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?model=whisper-large&language=fr&smart_format=true&punctuate=true&diarize=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

00b4f34f-fdff-4cac-bcdf-d7ee189c3867

### If possible, please attach your code or paste it into the text box.

    const transcription = await deepgram.listen.prerecorded.transcribeFile(
      fs.createReadStream(localVideoPath),
      {
        model: "whisper-large",
        language: language,
        smart_format: true,
        punctuate: true,
        diarize: true,
      }
    );

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@Marin-dev Our [Whisper Cloud](https://developers.deepgram.com/docs/models-languages-overview#deepgram-whisper-cloud) model will run much slower than our Nova and Nova -2 models. though the duration you are seeing seems like there might be something odd at play. 

Questions:

* Have you tried running this request using one of the Deepgram [Models](https://developers.deepgram.com/docs/model)? 
* Is there a specific reason you need to use Whisper here?
* If you must use Whisper try eliminating the use of the additional parameters first. For example if you use `smart_format` you don't also need to use `punctuate` as [Smart Forma](https://developers.deepgram.com/docs/smart-format)t does that for you. `Diarize` with Whisper can also be causing some issues here, so eliminating that from the request might help you pinpoint what might be causing the delays.

**Comment by SThiruNarayan** [comment]
Hello, 

We just subscribed to your service.
Could you please let me know which model corresponds to the Whisper Cloud hosted on Deepgram - https://developers.deepgram.com/docs/model? We would like to call use this model via the API.

Thanks
Narayan
